src/documentation/ToolkitDocumentation/CodeEmbed.tsx

```diff
@@ -115,35 +132,31 @@ interface CodePopUpProps extends BoxProps {
 // We draw the same code over the top in a portal so we can draw it
 // above the scrollbar.
 const CodePopUp = ({
-  setHovering,
   codeRef,
   concise,
   full,
+  background,
   ...props
 }: CodePopUpProps) => {
   // We need to re-render, we don't need the value.
   useScrollTop();
   useSplitViewContext();
-  const handleMouseEnter = useCallback(() => {
-    setHovering(true);
-  }, [setHovering]);
-  const handleMouseLeave = useCallback(() => {
-    setHovering(false);
-  }, [setHovering]);
-
   if (!codeRef.current) {
     return null;
   }
+
   return (
     <Portal>
       <Code
-        onMouseEnter={handleMouseEnter}
-        onMouseLeave={handleMouseLeave}
         concise={concise}
         full={full}
         position="absolute"
         top={codeRef.current.getBoundingClientRect().top + "px"}
         left={codeRef.current.getBoundingClientRect().left + "px"}
+        // We're always "raised" as this is the pop-up.
+        background="#E9F6F5"
+        boxShadow="rgba(0, 0, 0, 0.18) 0px 2px 6px"
```

Comment:
This is what I was ideally trying to avoid, because now there is no transition of background colour on mouseEnter on the `<Code>` component like there is on hovering the `insert code` button, or code fragments in the reference toolkit. There is, however, a transition on mouseLeave. I don't think it's that important, but it was why my code was a bit 'messy' before.

Reply:
Hah, I did go back and try to work out if I'd lost transitions but couldn't see it. I agree the transition is desirable, I'll have another go.

Reply:
We chatted about this and concluded that it looked good without this transition. I'd not seen it with the old code as I think it relied on rendering between certain setState calls.